website/docs/components/modal/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,352 @@
+  </M.Body>
+  <M.Footer>
+    <Hds::ButtonSet>
+      <Hds::Button type="submit" @text="Confirm" />
+      <Hds::Button type="button" @text="Cancel" @color="secondary" />
+    </Hds::ButtonSet>
+  </M.Footer>
+</Hds::Modal>
+
+A **warning** Modal indicates that the Modal's action may impact a setting, item, or object and is used to message the potential impact while confirming that the user wants to continue.
+
+This action may impact areas of the application outside of the scope of the current feature or function, but can also be tied to the context that the user is currently in. Common examples include:
+
+- disabling or enabling an application-wide setting or feature
+- archiving an item that can be recovered
+- changing a setting that may require the user to re-authenticate or perform an action again.
+
+### Critical
+
+<Hds::Modal @size="medium" @color="critical" open as |M|>
+  <M.Header as |H|>
+    Critical Modal
+  </M.Header>
+  <M.Body>
+    <p class="hds-typography-body-300 hds-foreground-primary">Modal content</p>
+  </M.Body>
+  <M.Footer>
+    <Hds::ButtonSet>
+      <Hds::Button type="submit" @text="Confirm" />
+      <Hds::Button type="button" @text="Cancel" @color="secondary" />
+    </Hds::ButtonSet>
+  </M.Footer>
+</Hds::Modal>
+
+A **critical** Modal is used to indicate a destructive action that is irreversible and will impact other settings, items, or objects within the current feature or in other areas of the application. Common examples include:
+
+- deleting an item or object that cannot be recovered
+- modifying a setting that cannot be changed or reversed in the future
+- alerting the user of unsaved changes that will be discarded.
+
+## Title icon
+
+### Best practices
+
+- A **titleIcon** can help reinforce the severity, strength, and importance of the **warning** and **critical** color variants.
+- The primary action in the **critical** variant should use a critical action button variant.
+
+## Modal body
+
+### Body type
+
+![Modal body types](/assets/components/modal/modal-body-type.png)
+
+- The **default** body type is meant for simple text-based content and could be used for a quick confirmation of an action or disclosure of information.
+- The **custom** body type is suitable for the majority of use cases when collecting information or feedback from the user. It can accept any custom component and nested components by swapping out the placeholder instance with a local component or relevant HDS component.
+
+## Modal footer
+
+### Actions
+
+#### General best practices
+
+- To main consistency the only property that should be changed in the actions within the footer is the text property. An exception to this is in the case of a Modal with critical or destruction function, in which the primary button should be changed to **critical**.
+- Action buttons should remain consistent with the variants defined in the number of actions in the footer; **primary**, **secondary**, and **tertiary**, in that order.
```

Comment:
Unsure if we have explicit guidance, but for me I've been using code blocks for attributes that can be passed in